pilot/docker/Dockerfile.proxy_debug

```diff
@@ -11,10 +11,5 @@ COPY envoy_mixer.json      /etc/istio/proxy/envoy_mixer.json
 COPY envoy_mixer_auth.json /etc/istio/proxy/envoy_mixer_auth.json
 COPY envoy_bootstrap_tmpl.json /var/lib/istio/envoy/envoy_bootstrap_tmpl.json
 
-# Sudoers used to allow tcpdump and other debug utilities.
-RUN useradd -m --uid 1337 istio-proxy && \
```

Comment:
why is this removed?

Reply:
Bad merge I guess. Will add it back.

I split the PR in 2 parts, in the other PR I was asked to move it, now merge seems to have missed that.